README.md

```diff
@@ -1,16 +1,26 @@
 # goharbor-client
 
-#### This project is still under development and not stable yet - breaking changes may happen at any time and without notice
+![Compile & Test](https://github.com/mittwald/goharbor-client/workflows/Compile%20&%20Test/badge.svg)
+[![GitHub license](https://img.shields.io/github/license/mittwald/goharbor-client.svg)](https://github.com/mittwald/goharbor-client/blob/master/LICENSE)
+[![Go Report Card](https://goreportcard.com/badge/github.com/mittwald/goharbor-client)](https://goreportcard.com/report/github.com/mittwald/goharbor-client)
+[![Documentation](https://godoc.org/github.com/mittwald/goharbor-client?status.svg)](https://pkg.go.dev/github.com/mittwald/goharbor-client)
 
-A Harbor API client enabling Go programs to perform CRUD operations on [goharbor](https://github.com/goharbor/harbor) users and projects
+<hr />
+**:warning: STABILITY NOTICE:** This project is still under development and not stable yet - breaking changes may happen at any time and without notice
```

Comment:
Seems like Markdown ignores the outlying '**' when there's an `:emojie:` inside.
https://github.com/martin-helmich/goharbor-client/tree/more-badges
```suggestion
:warning: **STABILITY NOTICE:** This project is still under development and not stable yet - breaking changes may happen at any time and without notice
```

Reply:
Meh. Thanks, markdown. 😞